en/peripherals/uavcan_escs.md

```diff
@@ -107,6 +129,19 @@ This assigns the following Sapog configuration parameters for each enumerated ES
 
 > **Note** See [Sapog reference manual](https://files.zubax.com/products/io.px4.sapog/Sapog_v2_Reference_Manual.pdf) for more information about the parameters.
 
+### Myxa ESC Setup
+
+Motor enumeration for Myxa [Telega-based ESCs](https://zubax.com/products/telega) is usually performed using the [Kucher tool](https://files.zubax.com/products/com.zubax.kucher/) (or less "GUI-friendly" [UAVCAN GUI Tool](https://uavcan.org/GUI_Tool/Overview/)).
+
+There is some guidance here: [Quick start guide for Myxa v0.1](https://forum.zubax.com/t/quick-start-guide-for-myxa-v0-1/911) (Zubax blog)
+
+ (e.g Mitochondrik and Myxa) you can use the .
```

Comment:
The what? :)

Reply:
There goes my aura of perfection. Thanks!